helm/overrides/local.yaml

```diff
@@ -3,7 +3,7 @@
 ### CFGOV OVERRIDES ###
 image:
   repository: cfgov_python
-  pullPolicy: Never
+  pullPolicy: IfNotPresent
```

Comment:
`cfgov_python` is only ever local. It can never be pulled to begin with. Switch this back to `Never` or it could try to reach out to Docker Hub.